src/sentry/eventstream/base.py

```diff
@@ -69,6 +70,10 @@ def start_delete_tag(self, project_id, tag):
     def end_delete_tag(self, state):
         pass
 
+    def requires_post_process_forwarder(self):
+        return False
```

Comment:
I tried to think if this could be implemented in a way that's a little easier to keep synchronized with whether or not `run_post_process_forwarder` has been defined, but I couldn't think of a clean way without encapsulating the forwarder as a separate object. (In that case, we could then check if the forwarder implementation is either defined or not on the backend class.) I don't really want to go that far right now, but if anybody has any clever ideas I'm open to them.